src/pages/[scheduleId]/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { MainLayout } from '@/components/layout/MainLayout'
+import { MemoList } from '@/components/screen/MemoList'
+import { USERS, SCHEDULE } from '@/constant/data'
+
+export default function Home() {
+  return (
+    <MainLayout>
+      <div className='m-auto mt-20 w-4/5 md:gap-4 md:grid'>
```

Comment:
全体としてgirdを使うようなレイアウトではないかなと思います
縦に等間隔に並べたいのであればflexで十分です

Reply:
全体中央揃え, 縦等間隔なら
```suggestion
      <div className='mx-auto mt-20 flex w-4/5 flex-col items-center gap-8'>
```
がちょうどいい塩梅なんじゃないかなと思います
flex-col: flexを縦並びにするやつ
items-center: 並べた方向の軸を中央に持ってくるやつ